packages/cosmic-swingset/economy-template.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "module": "@agoric/run-protocol/scripts/init-core.js",
+    "args": [
+      {
+        "econCommitteeOptions": {
+          "committeeSize": 1
+        }
+      }
+    ]
+  },
+  "@agoric/pegasus/scripts/init-core.js",
+  {
+    "module": "@agoric/run-protocol/scripts/add-collateral-core.js",
+    "args": [
```

Comment:
You were planning to make `entrypoint` explicit. Not yet, though?

Reply:
> You were planning to make `entrypoint` explicit. Not yet, though?

Ah yes.  Will do now.